### PR TITLE
network: radv - Allow to send hop limit

### PR DESCRIPTION
**Pull request by ssahani**

### CODE REVIEW

File: man/systemd.network.xml [hunk 1]
```diff
@@ -3089,6 +3089,14 @@ Token=prefixstable:2002:da8:1::</programlisting></para>
         for details. Defaults to <literal>medium</literal>.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>HopLimit=</varname></term>
+        <listitem>
+          <para>Configures IPv6 router hop limit. Takes an integer in the range 1…255. See also
```
Comment (yuwata) [inline man/systemd.network.xml]: `IPv6 router hop limit` sounds spurious. How about to simply `hop limit` or `IPv6 hop limit` ?

Also, the parser also accepts 0.

File: src/network/networkd-network-gperf.gperf [hunk 2]
```diff
@@ -371,6 +371,7 @@ IPv6SendRA.RouterLifetimeSec,                config_parse_router_lifetime,
 IPv6SendRA.Managed,                          config_parse_bool,                                        0,                             offsetof(Network, router_managed)
 IPv6SendRA.OtherInformation,                 config_parse_bool,                                        0,                             offsetof(Network, router_other_information)
 IPv6SendRA.RouterPreference,                 config_parse_router_preference,                           0,                             0
+IPv6SendRA.HopLimit,                         config_parse_router_hop_limit,                            0,                             0
```
Comment (yuwata) [inline src/network/networkd-network-gperf.gperf]: `config_parse_uint8` can be used.

**Review by yuwata — CHANGES_REQUESTED** [review]

File: man/systemd.network.xml (hunk 1 above)
Comment (ssahani) [reply man/systemd.network.xml]: Yes updated.

**Comment by ssahani** [comment]
Updated thanks for the review

**Review by yuwata — APPROVED** [review]

**Comment by yuwata** [comment]
Please rebase.

**Pull request merged**